visualization/create_charts.py

```
@@ -0,0 +1,366 @@
+
+        if 'final' in results:
+            stages.append('Multi-Agent')
+            accuracies.append(results['final']['accuracy'] * 100)
+            costs.append(cost_map['final'])
+
+        # 플롯 생성
+        plt.figure(figsize=(10, 6))
+        plt.scatter(costs, accuracies, s=200, alpha=0.6, c=range(len(stages)), cmap='viridis')
+
+        # 라벨 표시
+        for i, (cost, acc, stage) in enumerate(zip(costs, accuracies, stages)):
```

Ruff (0.14.13)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)